libnavigation-ui/src/main/java/com/mapbox/navigation/ui/route/MapRouteProgressChangeListener.kt

```diff
@@ -116,4 +131,9 @@ internal class MapRouteProgressChangeListener(
             (routeProgress.distanceRemaining + routeProgress.distanceTraveled)
         return routeProgress.distanceTraveled / totalDist
     }
+
+    private fun getRestoreArrowVisibilityFun(isVisible: Boolean): DeferredRouteUpdateFun = {
+        routeArrow.updateVisibilityTo(isVisible)
+    }
 }
+internal typealias DeferredRouteUpdateFun = () -> Unit
```

Comment:
Adding this caused the `MapRouteProgressChangeListenerKt` addition in `current.txt` but it's empty as all the APIs are `internal`. In any case, adding is not SemVer

> We might need to update / push `api/current.txt` files after running `$> make 1.0-core-update-api` (Core) / `$> make 1.0-ui-update-api` (UI) if there are changes / errors we're 🆗 with (e.g. `AddedMethod` changes are marked as errors but don't break SemVer) 🚀 If there are SemVer breaking changes add the `SEMVER` label. See [Metalava](https://github.com/mapbox/mapbox-navigation-android/blob/master/docs/metalava.md) docs